src/app/shared/models/notification-handler.ts

```diff
@@ -15,10 +23,18 @@ export interface SingleNotification {
   text?: string
   vibrate?: boolean
   sound?: boolean
+  id?: number
 }
 
 export interface NotificationTaskInfo {
   name: string
   timestamp: number
   completionWindow: number
 }
+
+export enum NotificationMessagingType {
+  LOCAL = 'LOCAL',
+  FCM = 'FCM',
```

Comment:
What is this for ?

Reply:
It's the enum for determining what notification service to use (local or fcm, etc) or do you mean why is `FCM` still there when there's `FCM_REST` and `FCM_XMPP`?

Reply:
Yes why FCM is still there when there are FCM_REST and FCM_XMPP?

Reply:
Oh cause the previous/current accepted input for the messaging type is `FCM`/`LOCAL` in the defaultConfig so I thought for backwards compatibility we can keep it, but since it's hardcoded anyway I can just remove it?

Reply:
If we are still using the same implementation for both FCM and FCM_XMPP then, there is no point of keeping both. I would remove one and apply the correct type where necessary.